README.md

```diff
@@ -29,9 +29,194 @@ This library is written in modern JavaScript and is published in both CommonJS a
 ## Usage
 
 ```js
+import { generateKeyPair, getKeyPairFromMnemonic, getKeyPairFromSeed } from 'human-crypto-keys';
```

Comment:
I think we are missing explaining exactly how the RSA and ED25519 keys are generated. For instance, we must explain that we are using https://github.com/indutny/hmac-drbg as the PRNG.